doc/howto/instances_manage.md

```diff
@@ -76,3 +76,21 @@ There are two ways to prevent accidental deletion of instances:
 
 - To protect a specific instance from being deleted, set [`security.protection.delete`](instance-options-security) to `true` for the instance.
   See {ref}`instances-configure` for instructions.
+
+### Rebuild an instance
+
+If you want to keep the same instance configuration, but you want to freshly start with a new instance content, you have the option to rebuild it.
+
+Once you have stopped your instance and that you ensured that the instance has no snapshots, you can proceed with the rebuild operation. It can be done in three different ways:
```

Comment:
Why would I check for snapshots? There's some information missing here.
Add something like "You can only rebuild instances that have no snapshots, because ..."